tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -142,8 +144,12 @@ spec:
                 echo "Error: Source container ${sourceContainer} not found!"
                 exit 1
               fi
-              sourceTag="${prefixedTag}-source"
-              push_image "${name}" "${sourceContainer}" "${repository}" "${sourceTag}"
+              if [ -z "$tagPrefix" ]; then
+                echo "Error: tagPrefix needs to be set when pushing source containers"
+                exit 1
+              fi
+              push_image "${name}" "${sourceContainer}" "${repository}" "${tagPrefix}-${timestamp}-source"
+              push_image "${name}" "${sourceContainer}" "${repository}" "${tagPrefix}-source"
```

Comment:
Question: Do we need this? Or should we still push using the timestamped tag only?

Reply:
In the classic pipeline, only the equivalent of `${tagPrefix}-${timestamp}-source` is tagged. I don't see evidence of a source container tag for the floating tag in the registry.

Reply:
OK, thanks, removed.